bool wingAsVirtualCenter
float64 waypointRadius
---
bool success
